.github/README_es.md

```diff
@@ -240,17 +240,17 @@ func main() {
 <details>
   <summary>📚 Mostrar más ejemplos de código</summary>
 
-### Views engines
+### Motores de Vistas
 
-📖 [Config](https://docs.gofiber.io/api/fiber#config)
-📖 [Engines](https://github.com/gofiber/template)
-📖 [Render](https://docs.gofiber.io/api/ctx#render)
+📖 [Configuración](https://docs.gofiber.io/api/fiber#config)
+📖 [Motores](https://github.com/gofiber/template)
+📖 [Rasterizado](https://docs.gofiber.io/api/ctx#render)
```

Comment:
```diff
- Rasterizado
+ Renderizado
```
"Rasterizado" is not the same as "Renderizado", which would be correct in this case.